### PR TITLE
feat: adding samples for default values

### PR DESCRIPTION
**Pull request by m-strzelczyk**
Featuring the usage report API.

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-compute/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary) -- not yet

### CODE REVIEW

File: samples/snippets/sample_default_values.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            to showcase default values behaviour.
+    """
+    usage_export_location = compute_v1.UsageExportLocation({
+        'bucket_name': bucket_name,
+        'report_name_prefix': report_name_prefix
+    })
+
+    if not report_name_prefix:
+        # Sending empty value for report_name_prefix, will result with the
+        # next usage report generated will have the default prefix value
```
Comment (rsamborski) [inline samples/snippets/sample_default_values.py:52]: Please consider changing to (it seems more grammatically correct to me):
"Sending empty value for report_name_prefix, will result the next usage report generated to have the default prefix value "usage_gce"."

File: samples/snippets/sample_default_values.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        project_id: project ID or project number of the project to update.
+    """
+    projects_client = compute_v1.ProjectsClient()
+    export_location = compute_v1.UsageExportLocation(
+        {'bucket_name': '', 'report_name_prefix': ''})
+
+    # Updating the setting with empty bucket name will disable the
+    # usage report generation.
+    projects_client.set_usage_export_bucket(
+        project=project_id, usage_export_location_resource=export_location)
```
Comment (rsamborski) [inline samples/snippets/sample_default_values.py:112]: Please double check if this actually works. I've noticed that for PHP you have to send 'null' as usage_export_location_resource. Setting empty bucket_name has no effect on the GCP setting.

File: samples/snippets/sample_default_values.py (hunk 1 above)
Comment (m-strzelczyk) [reply samples/snippets/sample_default_values.py:52]: OK

File: samples/snippets/sample_default_values.py (hunk 2 above)
Comment (m-strzelczyk) [reply samples/snippets/sample_default_values.py:112]: Hey, you're right! I was assuming that [the docs](https://cloud.google.com/compute/docs/logging/usage-export#disabling_usage_reports) are correct - turns out they aren't! Thanks for this catch.

File: samples/snippets/sample_default_values.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    """
+    usage_export_location = compute_v1.UsageExportLocation({
+        'bucket_name': bucket_name,
+        'report_name_prefix': report_name_prefix
+    })
+
+    if not report_name_prefix:
+        # Sending empty value for report_name_prefix, will result with the
+        # next usage report generated will have the default prefix value
+        # "usage_gce". (ref: https://cloud.google.com/compute/docs/reference/rest/v1/projects/get)
```
[the remark this answers is not in the document]
Comment (m-strzelczyk) [reply samples/snippets/sample_default_values.py:53]: Done.

File: samples/snippets/sample_default_values.py [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+        print("Setting report_name_prefix to empty value causes the report "
+              "to have the default prefix of `usage_gce`.")
+
+    projects_client = compute_v1.ProjectsClient()
+    operation = projects_client.set_usage_export_bucket(
+        project=project_id, usage_export_location_resource=usage_export_location)
+
+    op_client = compute_v1.GlobalOperationsClient()
+    op_client.wait(project=project_id, operation=operation.name)
+    return
```
Comment (busunkim96) [inline samples/snippets/sample_default_values.py:63]: I think it is more typical for samples to omit bare `return` statements. See https://cloud.google.com/speech-to-text/docs/samples/speech-quickstart for example.

File: samples/snippets/sample_default_values.py [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+
+# [START compute_usage_report_get]
+def get_usage_export_bucket(project_id: str) -> compute_v1.UsageExportLocation:
+    """
+    Retrieve Compute Engine usage export bucket for the Cloud project.
+    Replaces the empty value returned by the API with the default value used
+    to generate report file names.
+
+    Params:
```
Comment (busunkim96) [inline samples/snippets/sample_default_values.py:74]:
```suggestion
    Args:
```
Please follow https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings for docstring style.

File: samples/snippets/sample_default_values.py [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+        project_id: project ID or project number of the project to update.
+        bucket_name: Google Cloud Storage bucket used to store Compute Engine
+            usage reports. An existing Google Cloud Storage bucket is required.
+        report_name_prefix: Prefix of the usage report name which defaults to an empty string
+            to showcase default values behaviour.
+    """
+    usage_export_location = compute_v1.UsageExportLocation({
+        'bucket_name': bucket_name,
+        'report_name_prefix': report_name_prefix
+    })
```
Comment (busunkim96) [inline samples/snippets/sample_default_values.py:48]:
```suggestion
    usage_export_location = compute_v1.UsageExportLocation(
        bucket_name=bucket_name,
        report_name_prefix=report_name_prefix
    )
```

Constructing it this way allows for IDE code completion to help out.

**Review by busunkim96 — CHANGES_REQUESTED** [review]

File: samples/snippets/sample_default_values.py (hunk 4 above)
Comment (m-strzelczyk) [reply samples/snippets/sample_default_values.py:63]: OK.

File: samples/snippets/sample_default_values.py (hunk 5 above)
Comment (m-strzelczyk) [reply samples/snippets/sample_default_values.py:74]: OK.

File: samples/snippets/sample_default_values.py (hunk 6 above)
Comment (m-strzelczyk) [reply samples/snippets/sample_default_values.py:48]: OK

**Review by busunkim96 — APPROVED** [review]

**Pull request merged**